### PR TITLE
bug resolved:on changing email field button should be reset to notify me

### PR DESCRIPTION
**Pull request by mohitgadhavi1**
issue no : #3470 . it was a small bug  which resolve those issue where asking for email

### CODE REVIEW

**Comment by mohitgadhavi1** [comment]
I accidentally push to master branch (:

File: yarn.lock [hunk 1]
```diff
@@ -19987,10 +19987,10 @@ __metadata:
 
 "eslint-plugin-zeus-custom@file:./eslint-plugin-zeus-custom::locator=eslint-config-custom%40workspace%3Apackages%2Feslint-config-custom":
   version: 0.0.1
-  resolution: "eslint-plugin-zeus-custom@file:./eslint-plugin-zeus-custom#./eslint-plugin-zeus-custom::hash=a463b5&locator=eslint-config-custom%40workspace%3Apackages%2Feslint-config-custom"
```
Comment (hkirat) [inline yarn.lock:19990]: Can u revert this?

**Review by hkirat — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (mohitgadhavi1) [reply yarn.lock]: sure. this change has been committed.

File: yarn.lock [hunk 2]
```diff
@@ -19993,7 +19993,7 @@ __metadata:
   resolution: "eslint-plugin-zeus-custom@file:./eslint-plugin-zeus-custom#./eslint-plugin-zeus-custom::hash=a463b5&locator=eslint-config-custom%40workspace%3Apackages%2Feslint-config-custom"
   peerDependencies:
     eslint: ">=7.0.0"
-  checksum: ca4fd39d90debf3c06a3848687ddc99752f971754e2f249145b13677b08b7504d76a1aa54cb1a8bf5415835a2af8174eac5dc02733fb3b36a9e960d76b637179
+  checksum: 4e0331830d706a4ca5ab1743380087993e015437310bf5565a0fd885d8476bd2e5af942d3401bdc8c506b1536281f4358c1c9a17512ebb365bf359cfdb6180a6
```
Comment (hkirat) [inline yarn.lock:19996]: still seeing a change here

Comment (mohitgadhavi1) [reply yarn.lock]: ok. I have reverted this change also.

Comment (mohitgadhavi1) [reply yarn.lock]: I shouldnt have included this file in first place.

Comment (hkirat) [reply yarn.lock]: Look at the number of changes

**Pull request closed without merging**